enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/CoreGraphDatabase.java

```diff
@@ -57,4 +56,9 @@ public Role getRole()
     {
         return getDependencyResolver().resolveDependency( RaftMachine.class ).currentRole();
     }
+
+    public void stopCatchupServer()
+    {
+        ((EnterpriseCoreEditionModule) editionModule).stopCatchupServer();
```

Comment:
Is this safe?

Reply:
As safe as it was before. It previously relied upon resolveDependency( CatchupServer.class ). This is but a mere refactoring since CatchupServer.class no longer exists.